internal/output/plaintext_fns.go

```diff
@@ -40,29 +40,32 @@ var ErrorPlaintextOutputFn = func(r resource) string {
 	}
 }
 
-// MultipleEmailPlaintextOutputFn converts the resource to plain text specifically for member data.
-var MultipleEmailPlaintextOutputFn = func(r resource) string {
-	return fmt.Sprintf("* %s (%s)", r["email"], r["_id"])
-}
-
-// MultipleIDPlaintextOutputFn converts the resource to plain text for data without a key.
-var MultipleIDPlaintextOutputFn = func(r resource) string {
-	return fmt.Sprintf("* %s (%s)", r["name"], r["_id"])
-}
-
-// MultiplePlaintextOutputFn converts the resource to plain text based on its name and key in a list.
+// MultiplePlaintextOutputFn converts the resource to plain text.
```

Comment:
I refactored the check for which fields exist to show to one place so we only need one version of showing multiple resources.